worldedit-core/src/main/java/com/fastasyncworldedit/core/queue/implementation/chunk/ChunkHolder.java

```diff
@@ -998,6 +1024,13 @@ public synchronized <V extends IChunk> void init(IQueueExtent<V> extent, int chu
         this.extent = extent;
         this.chunkX = chunkX;
         this.chunkZ = chunkZ;
+        if (this.parentWrapper != null) {
+            try {
+                this.parentWrapper.invalidate(this);
+            } catch (IllegalStateException ignored) {
```

Comment:
Either it is valid for the invalidation to not succeed, in which case it shouldn't throw an exception but e.g., rather return a boolean, or we shouldn't silently ignore the exception.